### PR TITLE
frontend: Fix incorrect GraphQL query for default benchmark 

### PR DESCRIPTION
**Pull request by punchagan**
75c2bef360ab23d5480c4a1f10f1a8e5e890bd91 changed the default benchmark to have
a name instead of being NULL to be able to use Postgres's unique
constraints. But, the PR didn't fix the GraphQL queries correctly, which caused
the metrics to not be displayed in the UI, incorrectly.

### CODE REVIEW

File: frontend/src/App.res [hunk 1]
```diff
@@ -20,8 +20,9 @@ let makeGetBenchmarksVariables = (
   ~startDate,
   ~endDate,
 ): GetBenchmarks.t_variables => {
+  open AppHelpers
   let isMaster = Belt.Option.isNone(pullNumber)
-  let isDefaultBenchmark = Belt.Option.isNone(benchmarkName)
+  let benchmarkName = Belt.Option.isNone(benchmarkName) ? Some(defaultBenchmarkName) : benchmarkName
```
Comment (art-w) [inline frontend/src/App.res:25]: Is `benchmarkName` always `Some`? (can we remove the option, or is there still a usecase for `None`?)

**Comment by art-w** [comment]
Good catch! We should redeploy asap :)

**Review by art-w — APPROVED** [review]

File: frontend/src/App.res (hunk 1 above)
Comment (punchagan) [reply frontend/src/App.res:25]: Should we do this in a separate PR, since we'd like to redeploy this quickly?

Comment (punchagan) [reply frontend/src/App.res]: I also pushed a commit that changes the graphql schema to correctly mark the `benchmark_name` field as non-NULL which we changed in a recent fix/migration. Also, removed the unnecessary `option` for `benchmarkName`

Comment (punchagan) [reply frontend/src/App.res]: Updating the graphql_schema actually breaks a bunch of other type checks. Looking into it. :)

Comment (punchagan) [reply frontend/src/App.res]: Okay, this should be ready for another look, hopefully to be merged. :) Thanks for your patience, @art-w !

**Comment by punchagan** [comment]
The build failures on this PR are due to the lint job failing because of some `dune` issues. Should we merge and deploy this, @art-w ?

**Pull request merged**